test/view_component/translatable_test.rb

```diff
@@ -3,85 +3,87 @@
 require "test_helper"
 
 class TranslatableTest < ViewComponent::TestCase
-  def test_isolated_translations
-    render_inline(TranslatableComponent.new)
+  if Rails.version.to_f >= 5.2
```

Comment:
There is more work needed to make `Translatable` work with Rails < 5.2. I wanted to know if y'all think it's worth it to spend more time trying to support older Rails before tackling this.

Reply:
I think we should support up to 5.2 since that's the oldest currently supported version. If adding support for < 5.2 were easy, I'd say let's go for it. Since it's more difficult, I'm inclined to say no.